src/main/java/pageObjects/loginPage.java

```diff
@@ -0,0 +1,92 @@
+package pageObjects;
+
+import driver.DriverFactory;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+
+public class loginPage extends DriverFactory {
+
+    public loginPage(WebDriver driver) {
+        super(driver);
+
+        PageFactory.initElements(driver, this);
+    }
+    @FindBy(id = "username")
+    private  WebElement usernameField;
+
+    @FindBy(id = "password")
+    private  WebElement passwordField;
+
+    @FindBy (id = "log-in-button")
+    private  WebElement signInButton;
+
+    @FindBy(id="log-out")
+    private  WebElement logOut;
+    public void successfulSignInAsSuperAdmin() throws InterruptedException {
+        signInAsSuperAdmin();
+    }
+    public void successfulSignInAsAdmin() throws InterruptedException {
+        signInAsAdmin();
+    }
+    public void successfulSignInAsUser() throws InterruptedException {
+        signInAsUser();
+    }
+    public void logInNoUsername(){
+        driver.get("https://zenistock.zenitech.local/login");
+        usernameField.sendKeys("");
```

Comment:
if we send no username, shouldnt we just ignore the field since it is already empty by default?